includes/admin/class-wcmypa-admin.php

```diff
@@ -1188,15 +1188,16 @@ public function renderBarcodes(WC_Order $order): void
 
                 if (isset($lastOrder[self::META_PPS_UUID])) {
 
-                    echo $lastOrder[self::META_TRACK_TRACE] ? sprintf(
-                        '<a href="%s">%s</a>',
-                        TrackTraceUrl::create(
-                            $lastOrder[self::META_TRACK_TRACE],
-                            $order->get_shipping_postcode(),
-                            $order->get_shipping_country()
-                        ),
-                        $lastOrder[self::META_TRACK_TRACE]
-                    ) : esc_html(__('Concept created but not printed.', 'woocommerce-myparcel'));
+                    echo (isset($lastOrder[self::META_TRACK_TRACE]) && $lastOrder[self::META_TRACK_TRACE])
```

Comment:
Voor de zekerheid, krijg je niet alsnog een error als die key niet defined is? Voor zover ik zie checkt hij inderdaad of die key aanwezig is met isset(). Zo nee, stopt hij dan meteen of kijkt hij alsnog naar `$lastOrder[self::META_TRACK_TRACE`?

Reply:
De `&&` zorgt ervoor dat ie stopt bij de eerste statement die `false` is, omdat dan per definitie de hele statement `false` is. Bij `||` (or) zou hij de volgende ook gaan checken, omdat maar 1 dingetje waar hoeft te zijn van alle condities.